transitions/registration.js

```diff
@@ -333,14 +340,43 @@ module.exports = {
         var doc = options.doc,
             db = db || options.db;
 
-        transitionUtils.addUniqueId(db, doc, callback);
+        if (options.params.patient_id) {
+            var located = doc.fields[options.params.patient_id];
```

Comment:
`located` doesn't make much sense to me (feels like it should be a boolean). How about `providedId` or something?